openfe/protocols/openmm_septop/equil_septop_settings.py

```diff
@@ -258,6 +258,12 @@ def must_be_monotonic(cls, v):
 class SepTopEquilOutputSettings(MDOutputSettings):
```

Comment:
Thinking about this more - I really like this solution, i.e. each Protocol subclasses their own "equilibration settings". Mainly because they can all quack the same way, but to users these will look like unique settings objects.